addons/dragging/initKeybinds.sqf

```diff
@@ -0,0 +1,58 @@
+// Add Keybinds
+["ACE3 Common", QGVAR(drag), LLSTRING(DragKeybind), {
+    private _player = ACE_player;
+
+    if (!alive _player) exitWith {false};
```

Comment:
```suggestion
    if !(alive _player) exitWith {false};
```

Reply:
This is the opposite of the ACE Coding Guidelines (although I know both are used in the code)

https://ace3.acemod.org/wiki/development/coding-guidelines.html#55-brackets-around-code

Reply:
My rule of thumb is: If it's one statement that is easily readable (such as the above), I add it inside the brackets.

But then we have lines such as [these](https://github.com/acemod/ACE3/blob/52ed0fc6be5669521dd4acdd756e6afd18e4fe6d/addons/dragging/functions/fnc_canCarry.sqf#L21). I'm not sure how to interpret the ACE guidelines here - are these fine or should these types of statements be corrected?